### PR TITLE
KissAsian: Fix typo

### PR DESCRIPTION
**Pull request by TorchedSammy**

### CODE REVIEW

File: websites/K/KissAsian/presence.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ presence.on("UpdateData", async () => {
         }
       } else if (isNaN(duration)) {
         presenceData.startTimestamp = browsingStamp;
-        presenceData.details = "Looing at:";
+        presenceData.details = "Looikng at:";
```
Comment (ririxidev) [inline websites/K/KissAsian/presence.ts:90]: you made typo with another typo. It should be ``Looking``

**Review by ririxidev — CHANGES_REQUESTED** [review]
and bump version

File: websites/K/KissAsian/presence.ts (hunk 1 above)
Comment (Bas950) [reply websites/K/KissAsian/presence.ts:90]: introducing a new typo!

Comment (TorchedSammy) [reply websites/K/KissAsian/presence.ts:90]: :|

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**